samples/pmic/npmx/buck/README.rst

```diff
@@ -0,0 +1,101 @@
+.. _nPM1300 BUCK:
+
+nPM1300 BUCK
+############
+
+Overview
+********
+
+This sample demonstrates the usage of nPM1300 PMIC BUCKs.
+For testing connect voltage multimeter to VOUT1 (output voltage pin of BUCK1). In this example
+voltage can be changed only for BUCK1. BUCK2 voltage is set by default to 3.0V.
+In main.c file select your test case to be executed.
+You can also modify expected output voltages and selected GPIOs to control the system.
```

Comment:
This sentence should be the very first sentence in this file (line 6):
```suggestion
The BUCK sample demonstrates how to ... using nPM1300 PMIC.
```

Next sentences should be the first sentences of **Building and running** or **Testing** section - please decide where current testcases' descriptions should be placed and what is missing. Once done, I'll convert them into steps.
```suggestion

For testing purposes, do the following:

#. Connect a voltage multimeter to **VOUT1** that is the output voltage pin of BUCK1 mode.

.. note::
   In this sample, voltage can be changed only for BUCK1 mode.
   BUCK2 voltage is set by default to 3.0V.
   
#. Go to the main.c file and select your test case to be executed.

You can also modify expected output voltages and selected GPIOs to control the system.
```